### DISCUSSION TITLE
Can vector sink data to HDFS/Hadoop ?

### DISCUSSION THREAD

**Discussion by Rattnik** [vectordotdev/vector#7512] [Q&A] [answered]
Hi,
Now we use Flume to consume data from kafka and send it to HDFS. But we want use Vector for that task, because we do that in other cases and Vector is better than Flume.
Is it possible to do that?

**Comment by jszwedko** [comment, marked as answer]
Hi @Rattnik !

Would you want to use [WebHDFS](https://hadoop.apache.org/docs/r1.0.4/webhdfs.html) to write data to HDFS? I'm not super familiar with HDFS, but I see this is how [`fluentd` supports it](https://docs.fluentd.org/how-to-guides/http-to-hdfs).

**Reply by Rattnik** [reply]
Hi,
Yep, fluetnd supports it. but I wanted to know if it supports by Vector? If not, we will use Flume or fluentd.

Thanks!

**Reply by jszwedko** [reply]
Hi @Rattnik . Vector does not currently support it, but if WebHDFS would work for you, I'll open an issue.

**Reply by Rattnik** [reply]
Hi @jszwedko 
it will be very kind of you and we will gladly use the Vector for sync data to hdfs.
Thanks!!!

**Reply by jszwedko** [reply]
Opened issue here: https://github.com/timberio/vector/issues/7557#issue-899666111
